pkg/diagnostics/diagnostics.go

```diff
@@ -148,7 +120,7 @@ func getLsblk(debugPod *v1.Pod, o *clientsholder.ClientsHolder) (out interface{}
 }
 
 // getLspci gets lspci in Json format for a given node
-func getLspci(debugPod *v1.Pod, o *clientsholder.ClientsHolder) (out []string, err error) {
+func getLspci(debugPod *v1.Pod, o clientsholder.Command) (out []string, err error) {
```

Comment:
What about renaming getLspci to getHWTextOutput and passing the command to it to follow the pattern for  getHWJsonOutput ? This would be the way to store the output of any command that is not natively outputing Json.

Reply:
That works, I'll do that.